src/server/verification/cron/DisposeEnrollmentsPeriodic.js

```diff
@@ -0,0 +1,80 @@
+import moment from 'moment'
+import AdminWallet from '../../blockchain/AdminWallet'
+import UserDBPrivate from '../../db/mongo/user-privat-provider'
+import getTasksRunner from '../../cron/TaskRunner'
+import Logger from '../../../imports/logger'
+import Config from '../../server.config'
+
+import createEnrollmentProcessor from '../processor/EnrollmentProcessor'
+
+const logger = Logger.child({ from: 'disposeEnrollmentsPeriodic' })
+
+export class DisposeEnrollmentsPeriodicTask {
+  get schedule() {
+    return Config.enrollDisposalCron
+  }
+
+  get name() {
+    return 'DisposeEnrollmentPeriodic'
+  }
+
+  async execute({ setTime }) {
+    try {
+      logger.info('Starting worker', this.name)
+      const authenticationPeriod = await this.getAuthenticationPeriod()
+      logger.info('Authentication period found', authenticationPeriod)
+      const fvRecords = await this.getFVRecords(authenticationPeriod)
+      logger.info(`Fetched ${fvRecords.length} FV records from db`)
+      const disposeEnrollments = await this.disposeEnrollments(fvRecords)
+      logger.info(`Performed Dispose Enrollment for ${disposeEnrollments.length} enrollmentIds`)
+      const enrollmentIdentifiers = disposeEnrollments.reduce((arr, disposeEnrollment) => {
+        if (disposeEnrollment.status !== 'failed') arr.push(disposeEnrollment.enrollmentIdentifier)
+        return arr
+      }, [])
+      logger.info('Removing disposed FV Rcords from DB')
```

Comment:
log how many records are being delete
if records failed dispose log.warn number of records failed + their ids

Reply:
there is logging for this information at the disposeEnrollments method at EnrollmentProcessor class
if more logging is needed I will add

Reply:
add the number of records failed